VAMobile/src/screens/HealthScreen/SecureMessaging/SecureMessaging.tsx

```diff
@@ -91,6 +95,13 @@ const SecureMessaging: FC<SecureMessagingScreen> = ({ navigation }) => {
   const onTabUpdate = (selection: string): void => {
     const tab = selection as SecureMessagingTabTypes
     if (secureMessagingTab !== tab) {
+      if (tab === SecureMessagingTabTypesConstants.FOLDERS) {
+        _.forEach(folders, (folder) => {
```

Comment:
We should be able to use the built-in JS `forEach` function here, unless there's a reason we need to use underscore?

Reply:
I used the same code that used underscore for this section as the getListItemsForFolders method uses